CONTRIBUTING.md

```diff
@@ -989,10 +989,16 @@ Pull requests can be closed if:
 
 ### Repositories
 
+#### Naming
+
 * Project name must not contain odoo or openerp
 * Project name for localization is "l10n_belgium" for Belgium
```

Comment:
I'm seeing right now here a non related typo. It should be `l10n-belgium`. Can you change it?